### ISSUE TITLE
Loss value

### ISSUE THREAD

**Issue by Srutarshi** [shubham0204/Bounding_Box_Regression_TF#4]
@shubham0204 :- How much did your training and validation loss decrease in 100 epochs ?

When I use batch size = 3 and # epochs = 100, the training loss reduces from 1.1121 to 1.0060 and the validation loss oscillates between 1.0989 to 1.1028. 

Do you experience similar values ? I used learning_rate = 0.0001.

**Comment by shubham0204** [comment]
Which dataset are you using? Mention any modifications you've made.

**Comment by Srutarshi** [comment]
I have used the same "Image Localization Dataset" as you have mentioned in your Medium post https://towardsdatascience.com/getting-started-with-bounding-box-regression-in-tensorflow-743e22d0ccb3

I have not made any modifications in your code.

**Comment by shubham0204** [comment]
The training and the loss value depends on the random initialization of weights and biases in a CNN. You can try modifying the hyperparameters or even using feature extractors like VGG and Inception.
